packages/monnify/test/Adaptor.test.js

```diff
@@ -0,0 +1,123 @@
+import { expect } from 'chai';
+import { enableMockClient } from '@openfn/language-common/util';
+import testData from './fixtures.json' with { type: 'json' };
+import { request, post, get } from '../src/Adaptor.js';
+
+const testServer = enableMockClient('https://fake.monnify.com');
+
+const configuration = {
+  'baseUrl': 'https://fake.monnify.com',
+  'apiKey': 'MK_TEST_YRP3AJ7RQ2',
+  'secretKey': 'JKML52W7XQ43C02VRFDNQ8KLAU459W8C'
+}
+
+beforeEach(() => {
+  testServer
+    .intercept({
+      path: '/api/v1/auth/login',
+      method: 'POST',
+    })
+    .reply(200, testData.login.response);
+});
+
+describe('request', () => {
+  it('makes a successful GET request', async () => {
+    testServer
+      .intercept({
+        path: '/api/v2/disbursements/search-transactions?pageNo=0&pageSize=10&sourceAccountNumber=4864192954',
```

Comment:
Can we use the query object here instead of placing them in the url as a string?